src/v3/src/components/Form/Layout.tsx

```diff
@@ -26,17 +35,67 @@ import renderers from './renderers';
 // eslint-disable-next-line import/no-cycle
 import Stepper from './Stepper';
 
-type LayoutProps = {
-  uischema: UISchemaLayout;
-};
-
 const getElementKey = (element: UISchemaElement, index: number): string => {
   const defaultKey = [element.type, element.key, index].join('_');
   return element.type === 'Field' && (element as FieldElement).key
     ? [(element as FieldElement).key].join('_')
     : defaultKey;
 };
 
+type LayoutProps = {
+  uischema: UISchemaLayout;
+};
+
+type StyledAccordionProps = {
+  uischema: AccordionLayout;
+};
+
+const StyledAccordionSummary = styled((props: AccordionSummaryProps) => (
+  // eslint-disable-next-line react/jsx-props-no-spreading
+  <AccordionSummary {...props} />
+))(({ theme }) => ({
+  padding: 0,
+  width: 'fit-content',
+  '& .MuiAccordionSummary-content': {
+    // marginLeft: theme.spacing(1),
+    margin: 0,
+    color: theme.palette.primary.main,
+    '&:hover': {
+      textDecoration: 'underline',
+      textDecorationColor: theme.palette.primary.main,
+    },
+  },
+}));
+
+const Accordion: FunctionComponent<StyledAccordionProps> = ({ uischema }) => {
+  const { elements } = uischema;
+
+  return (
+    <Box>
+      {
+        elements.map((element) => (
+          <MuiAccordion
+            key={element.key}
+            disableGutters
+            elevation={0}
+          >
+            <StyledAccordionSummary
+              aria-controls={`${element.options.id}-content`}
+              id={`${element.options.id}-header`}
+            >
+              <Typography>{element.options.summary}</Typography>
+            </StyledAccordionSummary>
+            <AccordionDetails sx={{ padding: 0 }}>
+              {/* eslint-disable-next-line @typescript-eslint/no-use-before-define */}
+              <Layout uischema={element.options.content} />
+            </AccordionDetails>
+          </MuiAccordion>
+        ))
+      }
+    </Box>
+  );
+};
```

Comment:
Should I separate this out into its own file (similar to the Stepper)? The reason I left it here intially is because of the cyclic dependency lint issue that has to be disabled for the Stepper.

Reply:
I think this question will get answered here 😅 https://github.com/okta/okta-signin-widget/pull/2770#discussion_r965327360